cmd2/decorators.py

```diff
@@ -300,11 +300,16 @@ def cmd_wrapper(*args: Any, **kwargs: Dict[str, Any]) -> Optional[bool]:
             else:
                 # Add statement to Namespace and a getter function for it
                 setattr(ns, constants.NS_ATTR_STATEMENT, statement)
-                setattr(ns, 'get_statement', lambda: statement)
+                setattr(ns, 'get_statement', Cmd2AttributeWrapper(statement))
```

Comment:
Would `Cmd2NamespaceWrapper` be a better name?

Reply:
It wraps an attribute that `cmd2` adds to the `Namespace`. Since it's callable, I've considered `Cmd2AttributeGetter`.

Reply:
Yeah I like `Cmd2AttributeGetter` better than I do the `Wrapper` name

Reply:
After talking to @anselor, I'm gonna make a change so it's no longer callable. Instead the class will implement `__get__()` and `__set()`. This will function like a normal `Namespace` attribute but will still be a `Cmd2AttributeWrapper` type so you can easily filter them out.

Reply:
Nevermind. `__get__()` and `__set__()` only work on class attributes, not instance attributes. Since we aren't adding our `Cmd2AttributeWrappers` to the `Namespace` as class attributes, this won't work. The other approach is to add `get()` and `set()` methods to `Cmd2AttributeWrapper`.

Then a developer would use it this way:
```Python
# Get the handler
handler = args.cmd2_handler.get()
handler(args)

# Overwrite the handler
args.cmd2_handler.set(new_handler)
```